regulations/templates/regulations/comment-review-chrome.html

```diff
@@ -48,54 +43,99 @@ <h3 class="toc-type">Table of Contents</h3>
+                  <ul>
+                    <% _.each(comment.files, function(file) { %>
+                      <li class="group">
+                        <div class="filename">
+                          <%= file.name %>
+                        </div>
+                        <a href="<%= file.previewUrl %>" class="attachment-preview">Preview</a>
+                      </li>
+                    <% }) %>
+                  </ul>
+                </div>
+              <% } %>
+              <div class="edit-comment">
+                <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+                <svg width="14px" height="11px" viewBox="0 0 14 11" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                    <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+                        <g class="write-icon">
+                            <path d="M10.4,6.8 L10.4,8.1 C10.4,8.6 10.2,9.1 9.8,9.5 C9.4,9.9 9,10.1 8.4,10.1 L2.8,10.1 C2.3,10.1 1.8,9.9 1.4,9.5 C1,9.1 0.8,8.7 0.8,8.1 L0.8,2.5 C0.8,2 1,1.5 1.4,1.1 C1.8,0.7 2.2,0.5 2.8,0.5 L8.4,0.5 C8.7,0.5 8.9,0.6 9.2,0.7 C9.3,0.7 9.3,0.8 9.3,0.9 C9.3,1 9.3,1 9.2,1.1 L8.9,1.4 C8.8,1.5 8.8,1.5 8.7,1.5 L8.4,1.5 L2.8,1.5 C2.5,1.5 2.3,1.6 2,1.8 C1.8,2 1.7,2.3 1.7,2.6 L1.7,8.2 C1.7,8.5 1.8,8.7 2,9 C2.2,9.3 2.5,9.3 2.8,9.3 L8.4,9.3 C8.7,9.3 8.9,9.2 9.2,9 C9.5,8.8 9.5,8.5 9.5,8.2 L9.5,7.4 C9.5,7.3 9.5,7.3 9.6,7.3 L10,6.9 C10.1,6.8 10.1,6.8 10.2,6.9 C10.4,6.6 10.4,6.7 10.4,6.8 L10.4,6.8 Z M9.8,1.9 L11.7,3.8 L7.2,8.3 L5.3,8.3 L5.3,6.4 L9.8,1.9 L9.8,1.9 Z M6.9,7.6 L7.7,6.8 L6.7,5.8 L5.9,6.6 L5.9,7 L6.5,7 L6.5,7.6 L6.9,7.6 L6.9,7.6 Z M9.9,2.8 C9.8,2.7 9.8,2.7 9.7,2.8 L7.4,5.1 C7.3,5.2 7.3,5.2 7.4,5.3 C7.5,5.4 7.5,5.4 7.6,5.3 L9.9,3 C10,3 10,2.9 9.9,2.8 L9.9,2.8 Z M12.8,2.7 L12.2,3.3 L10.3,1.4 L10.9,0.8 C11,0.7 11.2,0.6 11.4,0.6 C11.6,0.6 11.7,0.7 11.9,0.8 L12.9,1.8 C13,1.9 13.1,2.1 13.1,2.3 C12.9,2.5 12.9,2.6 12.8,2.7 L12.8,2.7 Z" id="Shape"></path>
+                        </g>
+                    </g>
+                </svg>
+                <span>Edit this comment</span>
               </div>
-            <% } %>
+            </li>
+          <% }) %>
+        </ul>
+
+        <div class="download-comment group">
+          <h4>Download the text of your full comment as a PDF</h4>
+          <div class="details">
+            This does not include your attachments.
           </div>
-          <div>Edit comment</div>
-          <% if (index < comments.length - 1) { %>
-            <hr>
-          <% } %>
-        <% }) %>
+        </div>
+
       </div>
-      <div id="comment-preview">
-        <% if (!previewLoading) { %>
-          <button class="preview-button">Preview your comment</button>
-        <% } else { %>
-          <button class="preview-button" disabled>Building...</button>
-        <% } %>
+
+      <div class="additional-info">
+        <h4>Include your information (Optional)</h4>
```

Comment:
Something to ponder: this text is specific to EPA. Does it belong in `notice-and-comment`? Should it be a configuration? We can get the field list from regs.gov, but it doesn't quite match this, so there'd need to be some sort of conversion.